e2e-tests/package.json

```diff
@@ -15,14 +15,14 @@
     "lint": "eslint . --ext .js,.ts"
   },
   "devDependencies": {
-    "@cypress/eslint-plugin-dev": "^5.3.2",
+    "@cypress/eslint-plugin-dev": "5.3.2",
```

Comment:
intentional pinning to specific single versions instead of `^` versions?

Reply:
I think it's part of the way the version-bumping script works, isn't it @schultzp2020 ?

Reply:
That's been our rule in the showcase repo for a while. It must have been added while I was out.

Reply:
We recently [migrated from cypress to playwright](https://github.com/janus-idp/backstage-showcase/pull/851). And, this issue should be resolved when you [rebase](https://github.com/janus-idp/backstage-showcase/blob/main/e2e-tests/package.json).